kolibri/auth/models.py

```diff
@@ -1000,8 +1033,7 @@ def __str__(self):
 class Classroom(Collection):
 
     morango_model_name = "classroom"
-    _morango_proxy_order = 2
-
+    _morango_model_dependencies = (Facility,)
```

Comment:
If it's prefixed with `_`, it indicates that Morango intends this as internal use.. so would seem a modification of API is appropriate..

Reply:
its only used in the registration of syncable models which happens at app load time. It should not be used publicly anywhere else, hence the `_`. But if it is convention, then I can change it.

Reply:
Yeah, as we're using it from the Kolibri side, it's a "public" interface, and hence it would make sense not to have the `_` prefix. It's more like `morango_model_name`.